README.md

```diff
@@ -1,14 +1,75 @@
-# AzureIoT - Azure IoT library for Arduino
+# AzureIoT - Azure IoT Hub library for Arduino
 
-This library is a port of the [Microsoft Azure IoT device SDK for C](https://github.com/Azure/azure-iot-sdks/blob/master/c/readme.md) to Arduino. It allows you to use your Arduino/Genuino with the [Azure IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/).
+This library is a port of the [Microsoft Azure IoT device SDK for C](https://github.com/Azure/azure-iot-sdks/blob/master/c/readme.md) to Arduino. It allows you to use several Arduino compatible boards with Azure IoT Hub.
 
-For the Arduino/Genuino [MKR1000](https://www.arduino.cc/en/Main/ArduinoMKR1000) or [Zero](https://www.arduino.cc/en/Main/ArduinoBoardZero) and [WiFi Shield 101](https://www.arduino.cc/en/Main/ArduinoWiFiShield101) only.
+Currently supported hardware:
+- Atmel SAMD Based boards
+  - Arduino/Genuino [MKR1000](https://www.arduino.cc/en/Main/ArduinoMKR1000) (pending release)
+  - Adafruit [Feather M0](https://www.adafruit.com/products/3010)
+  - [Zero](https://www.arduino.cc/en/Main/ArduinoBoardZero) and [WiFi Shield 101](https://www.arduino.cc/en/Main/ArduinoWiFiShield101) 
+- ESP8266 based boards with [esp8266/arduino](https://github.com/esp8266/arduino)
+  - SparkFun [Thing](https://www.sparkfun.com/products/13711)
+  - Adafruit [Feather Huzzah](https://www.adafruit.com/products/2821)
 
 [![Microsoft Azure Certified][Microsoft-Azure-Certified-Badge]][azure-certifiedforiot]
 
-## Getting started
+## Prerequisites
 
-See the [Microsoft Azure IoT device SDK for C - Arduino WiFi Shield 101 and MKR1000 Setup Guide](https://github.com/Azure/azure-iot-sdks/blob/master/c/doc/run_sample_on_arduino_wifi101.md).
+You should have the following ready before beginning with any board:
+-   [Setup your IoT hub](https://github.com/Azure/azure-iot-sdks/blob/master/doc/setup_iothub.md)
+-   [Provision your device and get its credentials](https://github.com/Azure/azure-iot-sdks/blob/master/doc/manage_iot_hub.md)
+-   [Arduino IDE 1.6.8](https://www.arduino.cc/en/Main/Software) (currently this is available only as the hourly build)
+-  Install the AzureIoT library
+    * Currently this must be installed from source control by cloning the repo into your arduino libraries folder.  An example is below:
```

Comment:
Agreed, this was temporary as we had folks using it before it was merged, I will update this to reflect the library manager.